java/src/main/c++/vowpalWabbit_learner_VWLearners.cc

```diff
@@ -17,6 +17,18 @@ JNIEXPORT jlong JNICALL Java_vowpalWabbit_learner_VWLearners_initialize(JNIEnv *
   return vwPtr;
 }
 
+JNIEXPORT jlong JNICALL Java_vowpalWabbit_learner_VWLearners_seedVWModel(JNIEnv *env, jclass obj, jlong vwPtr)
+{ jlong cloneVwPtr = 0;
+  try
+  { vw* vwInstance = VW::seed_vw_model((vw*)vwPtr, "");
```

Comment:
ref: https://github.com/JohnLangford/vowpal_wabbit/blob/462ed1624967d99a349527edd108a6f62ed92b9a/vowpalwabbit/parse_args.cc#L1400-L1424

It [initializes new VW instances](https://github.com/JohnLangford/vowpal_wabbit/blob/462ed1624967d99a349527edd108a6f62ed92b9a/vowpalwabbit/parse_args.cc#L1416-L1417) using the parameters from an existing instance (seed), these cloned VW instances [shares the shared data with the the seed instance](https://github.com/JohnLangford/vowpal_wabbit/blob/462ed1624967d99a349527edd108a6f62ed92b9a/vowpalwabbit/parse_args.cc#L1419-L1421). 

The `seed_vw_model` method is expected to make the concurrent methods more memory efficient.